homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -96,7 +96,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         HOMEASSISTANT_DOMAIN,
         f"deprecated_yaml_{DOMAIN}",
         is_fixable=False,
-        breaks_in_ha_version="2024.10.0",
+        breaks_in_ha_version="2024.11.0",
```

Comment:
Shouldn't it be .12? 6 months deprecation

Reply:
Ah, I looked at media extractor which had .11